Update description for deny_null_bind on ldap auth backend config (#1043)
hashicorp/web-unified-docs: content/vault/v1.10.x/content/api-docs/auth/ldap.mdx

  username passed when authenticating. Examples: `sAMAccountName`, `cn`, `uid`
- `discoverdn` `(bool: false)` – Use anonymous bind to discover the bind DN of a
  user.

> - `deny_null_bind` `(bool: true)` – By default, Vault will prevent LDAP authentication 
>   attempts where the user provides an empty password (null binds). Setting this parameter 
>   to false allows Vault to support LDAP anonymous bind operations, which may be required 
>   for certain directory configurations that use anonymous search or discovery. When set to 
>   false, Vault defers the handling of empty-password authentication attempts to the LDAP 
>   server.

Reviewer: ```suggestion
- `deny_null_bind` `(bool: true)` – By default, Vault prevents LDAP authentication 
  attempts when the user provides an empty password (null binds). Setting
  `deny_null_bind` to `false` tells Vault to defer the handling of empty-password
  authentication attempts to the LDAP server. You may want to allow LDAP
  anonymous bind operations for directory configurations using anonymous search
  or discovery.
```
Style correction: write in active voice
